init/initialize.sh

```diff
@@ -97,6 +97,15 @@ sed -e "s;RSYNCDIR_SS;$RSYNCDIR_SS;" \
     -e "s;RSYNCDIR_NPAD;$RSYNCDIR_NPAD;" \
     -e "s;RSYNCDIR_PTR;$RSYNCDIR_PTR;" \
     $SLICEHOME/conf/rsyncd.conf.in > /etc/rsyncd.conf
+
+# Allow CIRA rsync access to the paris-traceroute module at their own sites
+for site in "yyz01 yyc01 yul01"; do
+    if echo $(hostname) | grep -q $site; then
+        HOSTS_ALLOW=$(grep 'hosts allow' $SLICEHOME/conf/rsyncd.conf.in)
+        echo "    ${HOSTS_ALLOW}, 192.211.124.208/28" > /etc/rsyncd.conf
```

Comment:
Also, does this mean that the extra `hosts allow` line will be at the end of the rsync config? If so, then it will only apply to the last module in the rsyncd.conf. Is that intended?